README.md

```diff
@@ -95,6 +105,53 @@ Status:
 Events:                      <none>
 ```
 
+## Example 2:
+
+creating healthcheck in a new namespace
+
+`kubectl create ns test`
+
+`kubectl create -f https://raw.githubusercontent.com/keikoproj/active-monitor/master/examples/inlineHello_ns.yaml`
+
+OR with local source code:
+
+`kubectl create -f examples/inlineHello_ns.yaml`
+
+Then, list all healthchecks:
+
+`kubectl get healthcheck -n test` OR `kubectl get hc -n test`
+
+```
+NAME                 AGE
+inline-hello-zz5vm   55s
+```
+
+View additional details/status of a healthcheck:
+
+`kubectl describe healthcheck inline-hello-zz5vm -n test`
+
+```
+...
+Status:
+  Failed Count:              0
+  Finished At:               2019-08-09T22:50:57Z
+  Last Successful Workflow:  inline-hello-4mwxf
+  Status:                    Succeeded
+  Success Count:             13
+Events:                      <none>
+```
+
+`argo list -n test`
+
+```                                                                                                            
+NAME                 STATUS      AGE   DURATION   PRIORITY
+inline-hello-88rh2   Succeeded   29s   7s         0
+inline-hello-xpsf5   Succeeded   1m    8s         0
+inline-hello-z8llk   Succeeded   2m    7s         0
+inline-hello-j48cn   Succeeded   3m    8s         0
```

Comment:
to save some scrolling in README, let's just cap the result set at 3, I think it conveys the same eitherway
```suggestion
```